### PR TITLE
Update cloudflare-pages.md for missing detail about changing package.json script

### PR DESCRIPTION
**Pull request by kleenkanteen**

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @kleenkanteen 

Basically, `$npm_execpath` will be replaced automatically with your package manager commands like `npm` or `yarn`. So, it's not necessary.

**Comment by kleenkanteen** [comment]
Didn't work for me though? I got an error. Am using git bash on w10

**Comment by kleenkanteen** [comment]
The error said $npm_execpath is not recognized also.

**Comment by yusukebe** [comment]
@kleenkanteen 

Did you run your app on Windows?

**Comment by kleenkanteen** [comment]
Yep. Here's a screenshot of the error:

image

I get the same error using powershell.

File: getting-started/cloudflare-pages.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ bun run dev
 
 ## 4. Deploy
 
-If you have a Cloudflare account, you can deploy to Cloudflare.
+If you have a Cloudflare account, you can deploy to Cloudflare. In `package.json` on line 7, replace `$npm_execpath` with your package manager of choice. Then run:
```
Comment (yusukebe) [inline getting-started/cloudflare-pages.md:124]: "line 7" may change, so delete it and just write "`$npm_execpath` to be changed".

**Comment by yusukebe** [comment]
@kleenkanteen 

Thanks for reporting. Indeed, `$npm_execpath` does not work well in some environments. So, your suggested description is necessary.

I've left one comment. Please check it.

**Comment by kleenkanteen** [comment]
Fixed

**Comment by yusukebe** [comment]
@kleenkanteen 

Thanks! I'll merge it now.

**Pull request merged**